src/pooled.rs

```diff
@@ -465,7 +465,10 @@ impl Quaint {
             Ok(conn) => conn,
             Err(mobc::Error::Timeout) => {
                 let state = self.inner.state().await;
-                return Err(Error::builder(ErrorKind::pool_timeout(state.max_open, state.in_use)).build());
+                let timeout_duration = self.pool_timeout.unwrap(); // We can use unwrap here because a pool timeout had to be set to get here
```

Comment:
It took me a second to understand the unwrap reason. You can't run into a timeout error without setting a timeout, so the timeout option must be set. Potentially rephrase it, but not a blocker.

Reply:
Cool. I've updated it a bit to make it clearer.